.travis.yml

```diff
@@ -23,21 +22,35 @@ script:
 
 matrix:
   include:
-    - env:
-        TESTNAME=quality
-        TARGETS="quality"
-    - env:
-        TESTNAME=test-python
-        TARGETS="test"
+    - python: 3.5
+      env:
+        TESTNAME=quality-python-3.5
+        TARGETS="PYTHON_ENV=py35 quality"
+    - python: 3.5
+      env:
+        TESTNAME=test-python-3.5
+        TARGETS="PYTHON_ENV=py35 test"
       after_success:
         - docker exec analytics_api_testing /edx/app/analytics_api/analytics_api/.travis/run_coverage.sh
         - codecov --disable pycov
     - env:
-        TESTNAME=test-python-django20
-        TARGETS="DJANGO_VERSION=django20 test"
-    - env:
-        TESTNAME=test-python-django21
-        TARGETS="DJANGO_VERSION=django21 test"
+        TESTNAME=test-python-3.5-django22
+        TARGETS="PYTHON_ENV=py35 DJANGO_VERSION=django22 test"
+
+    - python: 3.8
+      env:
+        TESTNAME=quality-python-3.8
+        TARGETS="PYTHON_ENV=py38 quality"
+    - python: 3.8
+      env:
+        TESTNAME=test-python-3.8
+        TARGETS="PYTHON_ENV=py38 test"
+      after_success:
```

Comment:
I am not sure, but I think that we should not run this at the same time for both versions of python.

Reply:
We need to add python3.8 support for tests too so how should we go about it then?

Reply:
The test will be running anyway, I mean that I think that we should remove the after_success section in order to upload the coverage only once to the codecov.